src/navigation/breadcrumbs/__test__/breadcrumbsMockups.js

```diff
@@ -0,0 +1,45 @@
+export default { // Router mockup
+    location: {
+        pathname: '/my-section/sub-section/some-status/cool-stuff',
+    },
+    params: {},
+    push: (to) => { alert(`it would push to: ${to}`); },
```

Comment:
Do we need to suppress ESLINT issue about alert here?
Honestly, do we need this?
We can always use a Jest mock function, and if we do need output, maybe `console.log()` (with accompanying suppression comment).

**_Not a Blocker_** from my perspective, but it does seem to me that having an alert doesn't seem super useful in a unit test mock.

Reply:
It got changed to a `console.log()` with a linter suppression comment.  Good enough.  I'll resolve this.